README.md

```diff
@@ -301,6 +301,45 @@ import exampleGo from '!!raw-loader!./code-example.go'
 ```
 ````
 
+### Mermaid graphs
+
+Use [mermaid.js](https://mermaid-js.github.io/mermaid/#/) to create graphs and
+diagrams,
+[for example](https://www.ory.sh/docs/kratos/self-service/flows/user-registration).
+The [mermaid live editor](https://mermaid.live) runs in the browser.  
+To use the graphs in documents import Mermaid at the top of the document:
+
+```js
+import Mermaid from "@site/src/theme/Mermaid"
+```
+
+Add the mermaid code where it should appear:
+
+```js
+<Mermaid
+  chart={`
+
+$Mermaid code goes here
+
+`}
+/>
+```
+
+for example:
+
+```js
+<Mermaid
+  chart={`
+graph TD
+    A[Christmas] -->|Get money| B(Go shopping)
+    B --> C{Let me think}
+    C -->|One| D[Laptop]
+    C -->|Two| E[iPhone]
+    C -->|Three| F[fa:fa-car Car]
+    `}
+/>
+```
```

Comment:
No point of giving an example with some dummy text and a proper diagram example. Please remove the example with dummy text.